compliance-monitor/monitor.py

```diff
@@ -569,7 +575,7 @@ async def get_detail(
         rows2, get_scopes(), grace_period_days=GRACE_PERIOD_DAYS,
         subjects=(subject, ), scopes=(scopeuuid, ),
     )
-    return render_view(VIEW_DETAIL, view_type, results2, base_url=settings.base_url, title=f'{subject} compliance')
+    return render_view(VIEW_DETAIL, view_type, results=results2, base_url=settings.base_url, title=f'{subject} compliance')
 
 
```

Comment:
wouldn't it be better to directly name this variable results instead of overwriting it?

Reply:
The variable can be renamed; the name was necessary in the past, but it isn't anymore.
However, the `results=results2` (or then `results=results`) will be necessary anyway, because it must be a keyword argument, so that `render_view` can propagate the argument to the call of `render` (line 556).